compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensionsOps.td

```diff
@@ -140,16 +140,26 @@ def ForeachThreadToWorkgroupOp : Op<Transform_Dialect,
     properties.
   }];
 
-  let arguments = (ins PDL_Operation:$target);
+  let arguments = (ins PDL_Operation:$target,
+                   Variadic<PDL_Operation>:$tile_sizes,
+                   DefaultValuedAttr<I64ArrayAttr, "{}">:$static_tile_sizes);
```

Comment:
Note: this is not the semantics of this op.
At this point in time, the tile sizes or num_threads have already been given by `transform.tile_to_foreach_thread`.
Passing it here will not result in the proper behavior.

I still owe you a transform dialect op to feed `transform.tile_to_foreach_thread` so that you can plug some of this logic, I'll send a separate PR.